pronto.gemspec

```
@@ -42,7 +42,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency('gitlab', '~> 4.4', '>= 4.4.0')
   s.add_runtime_dependency('httparty', '>= 0.13.7')
-  s.add_runtime_dependency('octokit', '~> 4.7', '>= 4.7.0')
+  s.add_runtime_dependency('octokit', '~> 5.0')
```

```suggestion
  s.add_runtime_dependency('octokit', '>= 4.7.0')
```
Maybe just remove the `~> 4.7` part?